docs/Sources/tuist/tuist.docc/Articles/Tuist/Users/sharing-code-across-manifests.md

```diff
@@ -0,0 +1,85 @@
+
+# Sharing code across manifests
+
+Project description helpers are a group of Swift files that can be accessed from any project manifest. They are useful to extract common patterns, define project conventions, or simplify the definition of projects.
+
+## Overview
+
+One of the inconveniences of Xcode when we use it with large projects is that it doesn't allow reusing elements of the projects other than the build settings through `.xcconfig` files. Being able to reuse project definitions is useful for the following reasons:
+
+- It eases the **maintenance** because changes can be applied in one place and all the projects get the changes automatically.
+- It makes it possible to define **conventions** that new projects can conform to.
+- Projects are more **consistent** and therefore the likelihood of broken builds due inconsistencies is significantly less.
+- Adding a new projects becomes an easy task because we can reuse the existing logic.
+
+Reusing code across manifest files is possible in Tuist thanks to the concept of **project description helpers**.
+
+### Definition
+
+Project description helpers are Swift files that get compiled into a framework, `ProjectDescriptionHelpers`, that manifest files can import.
+
+> Structure:
+Tuist is not opinionated about the files structure and the content in them, so it's up to you to give them a structure that makes sense for your project.
+
+You can import them into your manifest file by adding an import statement at the top of the file:
+
+```swift
+// Project.swift
+import ProjectDescription
+import ProjectDescriptionHelpers
+```
+
+### Location
+
+Tuist traverses up the directory hierarchy until it finds a `Tuist` directory. Then it builds the helpers module including all the files under the `ProjectDescriptionHelpers` directory in the `Tuist` directory.
```

Comment:
I'd adjust the "project structure" page to mention the concepts:
- Root directory: directory containing the Tuist directory
- Tuist directory: the `Tuist/` directory relative to the root.

Then you don't need to repeat ourselves here explaining the traversing logic that Tuist follows because it's the same across multiple-features.